### DISCUSSION TITLE
3000元装机和NAS升级

### DISCUSSION THREAD

**Discussion by giscus** [caibingcheng/comment#315] [Announcements]
# 3000元装机和NAS升级

家里已经有一台服役了三年的R3500X + RTX1660S的台式机，主要是女朋友在用。我的主力机是Acer Swift S3, i5 1240P CPU + 核显，日常刷视频、写写代码、写写文章是完全够用，但是如果想玩游戏或者想做点其他东西，性能就不够了，因此又组了一台机器，并且将NAS的机箱升级了一下。

https://imcbc.cn/202307/3k-diypc-nasup/

**Comment by penndu** [comment]
真心不是一般的省~

**Reply by caibingcheng** [reply]
省出来的可以吃好几顿排骨了~

**Reply by penndu** [reply]
额额，你这追求~
